routing/pathfind.go

```diff
@@ -1253,6 +1257,12 @@ func processNodeForBlindedPath(g Graph, node route.Vertex,
 		return nil, false, nil
 	}
 
+	// If we have explicitly been told to ignore this node for blinded paths
+	// then we skip it too.
+	if restrictions.nodeOmissionSet[node] {
```

Comment:
Do we also need to mark the node as visited to avoid repeatedly relaxing into it?

Reply:
good idea